### DISCUSSION TITLE
is there a reason for u8x8 not using open_iconic_all?

### DISCUSSION THREAD

**Discussion by Seitekful** [olikraus/u8g2#1394] [Q&A]
Hey, 
so I wonder why the open_iconic_all (and other open_iconic) fonts are not supportet for u8x8. I would love to use some other icons, but before I go ahead and struggle my way through the creation process (i have no clue about png2bdf/do_iconic.sh/linux) I want to know if these missing fontgroups are missing for a reason. Like they cant be processed because of some technical thing which lies outside my knowledge. Does anyone see a reason that they have been left out (besides memory) ?

**Comment by olikraus** [comment]
There is no reason as such. Which size do you need? Can you create an issue for this?

**Reply by Seitekful** [reply]
Thanks for your reply! Alright good to know, ill create an issue. I need the 1x1 and 2x2. Since I'm spared the hassle I would like to donate to you :) do you have a link or so?
